specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/FormRecognizer.json

```diff
@@ -0,0 +1,2344 @@
+                "valuePhoneNumber": {
+                    "description": "Phone number value.",
+                    "type": "string"
+                },
+                "valueNumber": {
+                    "description": "Floating point value.",
+                    "type": "number",
+                    "x-nullable": false
+                },
+                "valueInteger": {
+                    "description": "Integer value.",
+                    "type": "integer",
+                    "x-nullable": false
+                },
+                "valueArray": {
+                    "description": "Array of field values.",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/FieldValue"
+                    }
+                },
+                "valueObject": {
+                    "description": "Dictionary of named field values.",
+                    "type": "object",
+                    "additionalProperties": {
+                        "$ref": "#/definitions/FieldValue"
+                    }
+                },
+                "valueSelectionMark": {
+                    "description": "Selection mark value.",
+                    "type": "string",
+                    "enum": [
+                        "selected",
+                        "unselected"
+                    ],
+                    "x-nullable": false
+                },
+                "text": {
+                    "description": "Text content of the extracted field.",
+                    "type": "string"
+                },
+                "boundingBox": {
+                    "description": "Bounding box of the field value, if appropriate.",
+                    "$ref": "#/definitions/BoundingBox"
+                },
+                "confidence": {
+                    "description": "Confidence score.",
+                    "$ref": "#/definitions/Confidence"
+                },
+                "elements": {
+                    "description": "When includeTextDetails is set to true, a list of references to the text elements constituting this field.",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ElementReference"
+                    }
+                },
+                "page": {
+                    "description": "The 1-based page number in the input document.",
+                    "type": "integer",
+                    "minimum": 1,
+                    "x-nullable": false
+                }
+            }
+        },
+        "FieldValueType": {
+            "type": "string",
+            "description": "Semantic data type of the field value.",
+            "enum": [
+                "string",
+                "date",
+                "time",
+                "phoneNumber",
+                "number",
+                "integer",
+                "array",
+                "object",
+                "selectionMark"
```

Comment:
needs "gender", "country" for ID documents

Reply:
@ramparab Can you please add the following:

```
            "enum": [
                "string",
                "date",
                "time",
                "phoneNumber",
                "number",
                "integer",
                "array",
                "object",
                "selectionMark",
                "gender",
                "country"
            ],
```

Reply:
done